<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DrawOpenGL" name="OpenGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>241</width>
      <height>366</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Vertical Lines</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="verticalLinesSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Rotate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="x_slider_rotate">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="y_slider_rotate">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="z_slider_rotate">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Move</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSlider" name="x_slider_move">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSlider" name="y_slider_move">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSlider" name="z_slider_move">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>ORHTO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PERSPECTIVE</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QSlider" name="z_slider_look">
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QSlider" name="y_slider_look">
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QSlider" name="x_slider_look">
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Look at</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>turn on</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QSlider" name="zoom_slider_look">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>zoom</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawOpenGL</class>
   <extends>QOpenGLWidget</extends>
   <header>drawopengl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
